packages/mongodb-rag-ingest/src/embed/updateEmbeddedContent.test.ts

```diff
@@ -138,4 +148,139 @@ describe("updateEmbeddedContent", () => {
       "2cbfe9901657ca15260fe7f58c3132ac1ebd0d610896082ca1aaad0335f2e3f1"
     );
   });
+  it("processes chunks concurrently within a page", async () => {
+    const embeddedContentStore = makeMockEmbeddedContentStore();
+    const page: PersistedPage = {
+      ...examplePage,
+      updated: new Date(),
+      action: "updated",
+    };
+
+    // Spy and mock chunkPage
+    const chunkPageSpy = jest
```

Comment:
as an aside: i think this is a reasonable approach b/c we don't have the `chunkPage()` function as something that you can pass as config...but we really should (perhaps keeping this as the default).

can you see if we have a ticket for this, and if not, create one?